string name
humanoid_models/HumanoidJoint[] joints
